AGGREGATE.md

```diff
@@ -237,20 +233,25 @@ utilize techniques like retries to minimize data loss.
   How the browser obtains the service’s public key is left as an implementation
   detail.
 
-* The `privacy_budget_key` is used to define distinct batches of aggregate
-  reports. It is used by the aggregation service to prevent replay attacks. It
-  will be a hash of: `reporting_origin | source_site | destination | version`.
-  Note that the true key used to track batches will be `privacy_budget_key`
-  concatenated with `round_to_hour(scheduled_report_time)`. The latter is
-  omitted from the key for ergonomic reasons to help keys stay consistent over
-  time. All reports that share a privacy budget key should be sent to the aggregation service at the same time (in any order).
-
-
 * The `shared_info` will be a serialized JSON object. The authenticated data for
   decryption will consist of this string (encoded as UTF-8) with a constant
   prefix added for domain separation. The string therefore must be forwarded to
   the aggregation service unmodified. The reporting origin can parse the string
   to access the encoded fields.
+  
+* The `source_registration_time` within `shared_info` will represent (in seconds 
+  since the Unix Epoch) the time the source event was registered, rounded to the
+  nearest whole day.
+  
+* The `privacy_budget_key` is used to define distinct batches of aggregate
+  reports. It is used by the aggregation service to prevent replay attacks. It
+  will be a hash of: `reporting_origin | source_site | destination | version`.
+  Note that the true key used to track batches will be `privacy_budget_key`
+  concatenated with `round_to_hour(scheduled_report_time)` and 
+  `round_to_hour(source_registration_time`. The latter two are omitted from the key
```

Comment:
source_registration_time should already be rounded to the nearest day, so we can just use it directly